src/hooks/use-resource.ts

```diff
@@ -27,11 +28,11 @@ type UseResourceResponse<T> = {
 
 type UseResourceOptions<T> = {
   mode: 'PUT',
-  resource: Resource<T>,
+  resource: Resource<T> | string,
```

Comment:
Could you clarify how a resource would be a string in this context? Would it be a link/url?

Reply:
Yes, I think it would be handy in some cases to be able to just pass a url